core/web-assets/src/main/assets/js/apps/onms-classifications/views/group.html

```diff
@@ -108,12 +118,12 @@ <h5 class="pt-2">{{group.description}} <span class="badge badge-secondary" ng-if
                 <td class="col-md-1 text-nowrap">
                     <span ng-repeat="protocol in rule.protocols" class="badge badge-secondary mr-2">{{protocol}}</span>
                 </td>
-                <td class="col-md-2 text-nowrap">{{rule['srcAddress']}}</td>
-                <td class="col-md-1 text-nowrap">{{rule['srcPort']}}</td>
-                <td class="col-md-2 text-nowrap">{{rule['dstAddress']}}</td>
-                <td class="col-md-1 text-nowrap">{{rule['dstPort']}}</td>
-                <td ng-class="{'col-md-1': group.readOnly === false, 'col-md-2': group.readOnly === true}" class="text-nowrap">
-                    {{rule['exporterFilter']}}
+                <td class="col-md-2 wrapped">{{splitRule(rule['srcAddress'])}}</td>
+                <td class="col-md-1 wrapped">{{splitRule(rule['srcPort'])}}</td>
+                <td class="col-md-2 wrapped">{{splitRule(rule['dstAddress'])}}</td>
+                <td class="col-md-1 wrapped">{{splitRule(rule['dstPort'])}}</td>
+                <td ng-class="{'col-md-1': group.readOnly === false, 'col-md-2': group.readOnly === true}" class="text-ellipsis">
+                    <span title="{{rule['exporterFilter']}}">{{rule['exporterFilter']}}</span>
```

Comment:
What's the benefit of having the exact same value in the title as shown in the text?

Reply:
Because the CSS was modified to truncate it if it’s too long. This makes it easy to hover and see the full value. (See our discussion in the JIRA issue)

Reply:
Makes sense